src/menu/menu.h

```diff
@@ -7,9 +7,15 @@
 #ifndef MENU_H__
 #define MENU_H__
 
+#include <libdragon.h>
 
 #include "boot/boot.h"
 
+extern wav64_t sfx_cursor, sfx_error, sfx_enter, sfx_exit, sfx_settings;
```

Comment:
No need to extern that if loading is moved to the `sound.c` and playing is handled through wrapper function `sound_play_effect(sound_sfx_t sfx);`